### PR TITLE
CCITCARBON-273 Create a Sidebar for GitHub Pages Docs

### PR DESCRIPTION
**Pull request by shay6**

### CODE REVIEW

**Comment by shay6** [comment]
See the example on my repo
https://shay6.github.io/teflo_terraform_plugin/

**Review by JunqiZhang0 — DISMISSED** [review]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+nav_order: 1
+---
 
-This is Teflo project's plugin
+# Teflo Terraform Plugin
+{: .no_toc }
 
-- If you're a user of the plugin please refer to the [user](https://redhatqe.github.io/teflo_terraform_plugin/user.html) documentation.
-- If you're a maintainer/developer please refer to the [development](https://redhatqe.github.io/teflo_terraform_plugin/contribute.html) documentation.  
+This is Teflo project's plugin.
```
Comment (rujutashinde) [inline docs/index.md]: Please add the definition of this plugin 
This is a Teflo provisioner plugin to integrate with Terraform

File: docs/contribute.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+
+## Table of contents                                                            
+{: .no_toc .text-delta }                                                        
+
+1. TOC                                                                          
+{:toc}    
 
-### To return to documentation home page press [here](https://redhatqe.github.io/teflo_terraform_plugin/index.html)
 
 Hey Developers! If you want to contribute to this repository, please follow below steps:
```
Comment (rujutashinde) [inline docs/contribute.md:17]: I guess we can update the developers guide a bit better. but that can be a separate ticket

**Review by rujutashinde — CHANGES_REQUESTED** [review]

File: docs/contribute.md (hunk 2 above)
Comment (shay6) [reply docs/contribute.md:17]: yes, I didn't modify the original terraform docs.

File: docs/index.md (hunk 1 above)
Comment (shay6) [reply docs/index.md]: done

File: docs/contribute.md (hunk 2 above)
Comment (shay6) [reply docs/contribute.md:17]: will be fixed in a separate ticket

**Review by rujutashinde — APPROVED** [review]

**Pull request merged**